app/views/articles/index.html.erb

```diff
@@ -1,11 +1,5 @@
-<% @articles.each do |article| %>
-    <h1>
-        <%= article.title %>
-    </h1>
-    <p>
-        <%= article.lead %>
-    </p>
-    <p>
-        <%= article.content %>
-    </p>
-<% end %> 
\ No newline at end of file
+<h1> The Viking Times </h1>
+
+<% @articles.each do |article|%>
+    <%= link_to article.title, article_path(article) %>
```

Comment:
Are you only showing the article title on index page?

Reply:
fixed